services/provider/server/server.go

```diff
@@ -814,24 +816,36 @@ func (s *OCSProviderServer) GetStorageClaimConfig(ctx context.Context, req *pb.S
 					}),
 					Labels: getExternalResourceLabels("VolumeGroupSnapshotClass", mirroringEnabled, false, replicationID, storageID)},
 				&pb.ExternalResource{
-					Name: "ceph-rbd",
+					Name: req.StorageClaimName,
 					Kind: "VolumeReplicationClass",
-					Data: mustMarshal(map[string]string{
-						"replication.storage.openshift.io/replication-secret-name": provisionerSecretName,
-						"mirroringMode": "snapshot",
+					Data: mustMarshal(&replicationv1alpha1.VolumeReplicationClassSpec{
+						Parameters: map[string]string{
+							"replication.storage.openshift.io/replication-secret-name": provisionerSecretName,
+							"mirroringMode": "snapshot",
+							// This is a temporary fix till we get the replication schedule to ocs-operator
+							"schedulingInterval": "5m",
+							"clusterID":          clientProfile,
+						},
+						Provisioner: util.RbdDriverName,
 					}),
 					Labels: getExternalResourceLabels("VolumeReplicationClass", mirroringEnabled, false, replicationID, storageID),
 					Annotations: map[string]string{
 						"replication.storage.openshift.io/is-default-class": "true",
 					},
 				},
 				&pb.ExternalResource{
-					Name: "ceph-rbd-flatten",
+					Name: fmt.Sprintf("%s-flatten", req.StorageClaimName),
```

Comment:
Like all other classes, our VRCs should have a fixed name and not one that depends on the request